cutadapt/modifiers.py

```diff
@@ -227,12 +235,12 @@ def __call__(self, read):
 		read.qualities = read.qualities.translate(self.zero_cap_trans)
 		return read
 
-
-def PrimerTrimmer(read):
-	"""Trim primer base from colorspace reads"""
-	read = read[1:]
-	read.primer = ''
-	return read
+class PrimerTrimmer(object):
```

Comment:
Maybe this is necessary, but I need to know why. "Make PrimerTrimmer a class" is not a sufficient explanation ...